app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/tokens/CheckboxTokens.kt

```diff
@@ -0,0 +1,129 @@
+package com.hedvig.android.design.system.hedvig.tokens
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.design.system.hedvig.CheckboxDefaults.CheckboxStyle
+
+internal object CheckboxColorTokens {
+  val ContainerColor = ColorSchemeKeyTokens.SurfacePrimary
+  val OptionTextColor = ColorSchemeKeyTokens.TextPrimary
+  val LabelTextColor = ColorSchemeKeyTokens.TextSecondaryTranslucent
+  val DisabledOptionTextColor = ColorSchemeKeyTokens.TextDisabledTranslucent
+  val DisabledLabelTextColor = ColorSchemeKeyTokens.TextDisabledTranslucent
+  val ChosenIndicatorColor = ColorSchemeKeyTokens.SignalGreenElement
+  val NotChosenIndicatorColor = ColorSchemeKeyTokens.FillDisabled // todo: if i put borderSecondary here as it is in figma, it's almost invisible
```

Comment:
Aha the problem then is that the contrast with the background is just so low that it's unreasonably hard to see? Some of our "disabled" colors are like that, but is this even more so than the others?

Reply:
I think this is something I better show you. It's a bit confusing for me here